### PR TITLE
Organization pages

### PR DESCRIPTION
**Pull request by zwolf**
Adds pages to organizations, just like projects. This is mostly duplicated code from the projects version of pages. I would like to extract it all out and DRY it all up, but with only these two uses it seems a bit like a waste of effort. This was everyone's take when orgs were first dropped in (new controller scheme aside) and it's still true. When something comes along that can make use of this code again, I think that'll be the time.

# Review checklist

- [ ] First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
- [ ] If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
- [ ] If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
- [ ] Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

**Comment by camallen** [comment]
can you fix the hound warnings and leave the whitespace in the structure.sql (regenerate it via the migration task) as it's auto-generated so it'll have whitespace the next time we create a migration.

File: app/controllers/api/v1/organization_pages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    super create_params
+  end
+
+  def link_header(resource)
+    resource = resource.first
+    send(:"api_#{ resource_name }_url", id: resource.id, organization_id: resource.organization_id)
+  end
+
+  def set_language_from_header
+    params[:language] = "en"
```
Comment (camallen) [inline app/controllers/api/v1/organization_pages_controller.rb]: this is a straight copy but we should probably note that it's a working solution. we need this to actually respect the headers with a default back to english.

File: app/serializers/organization_page_serializer.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+class OrganizationPageSerializer
+  include RestPack::Serializer
```
Comment (camallen) [inline app/serializers/organization_page_serializer.rb]: can you use the restpack serializer wrapper here instead and setup preloading to avoid N+1 for linked resources.

File: app/serializers/organization_page_serializer.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  include CachedSerializer
+
+  attributes :id, :href, :created_at, :updated_at, :url_key, :title,
+    :language, :content, :type
+
+  can_include :organization
+  can_filter_by :url_key, :language
+
+  def self.page_href(page, options)
```
Comment (camallen) [inline app/serializers/organization_page_serializer.rb]: this is pretty much a straight copy coudn't we extract the code to a module that reflected on the resource class / id and just include that to get the behaviour? understand if you'd prefer not to

Comment (zwolf) [inline app/serializers/organization_page_serializer.rb]: That's true for a lot of this, as I said up top. Obviously it'd be better if all the duped code was cleaned up. This was faster, and short of adding another level of organization here we won't need it again. I can extract this all out for the serializers, controllers, etc., if you'd prefer.

Comment (zwolf) [inline app/serializers/organization_page_serializer.rb]: Might be better in a separate PR, to let the lab app devs use this in the meantime.

**Comment by zwolf** [comment]
Guess the projects code for same predates hound, fixed both.

File: app/serializers/organization_page_serializer.rb (hunk 3 above)
Comment (camallen) [inline app/serializers/organization_page_serializer.rb]: no problems.

File: db/structure.sql [hunk 4]
```diff
@@ -2568,14 +2632,14 @@ CREATE INDEX index_projects_on_migrated ON projects USING btree (migrated) WHERE
 
 
 --
--- Name: index_projects_on_mobile_friendly; Type: INDEX; Schema: public; Owner: -
+-- Name: index_projects_on_mobile_friendly; Type: INDEX; Schema: public; Owner: -; Tablespace:
```
Comment (camallen) [inline db/structure.sql]: Any ideas why these tablespace directives crept in? Not a blocker.

Comment (zwolf) [inline db/structure.sql]: Happened when I updated the structure, presumably because I had already updated my database from the master branch and a more recent migration. Should merge fine.

Comment (camallen) [inline db/structure.sql]: OOI what postgres server version is it?
